src/libstd/time.rs

```diff
@@ -60,6 +60,22 @@ pub use core::time::Duration;
 /// }
 /// ```
 ///
+/// # OS-specific behaviours
+///
+/// As `Instant` is a wrapper around system-specific types and it may behave
+/// differently depending on the underlying operating system.
+///
+/// The following snippet is fine on Linux but panics on MacOs.
```

Comment:
```suggestion
/// An `Instant` is a wrapper around system-specific types and it may behave
/// differently depending on the underlying operating system. For example, 
/// the following snippet is fine on Linux but panics on macOS:
```